agent/doc/swdesign/README.md

```diff
@@ -161,6 +161,461 @@ Needs:
 - utest
 - itest
 
+#### AgentManager shall receive workload states of the workloads it manages
```

Comment:
The most of the requirements' title is not in "shall" form, maybe we should stick to the simple present form for the title:
```suggestion
#### AgentManager receives workload states of the workloads it manages
```

Reply:
Done